ecole-doctorale-de-sciences-po-author-date-french.csl

```diff
@@ -0,0 +1,440 @@
+       <text variable="medium"/>
+</if>
+</choose>
+    </macro>
+<macro name="number">
+<choose>
+<if type="patent" match="any">
+       <text variable="number" prefix="Brevet "/>
+   </if>
+<else-if type="report" match="any">
+<text variable="number" prefix="Rapport "/>
+      </else-if>
+   </choose>
+</macro>
+<macro name="dimensions">
+<choose>
+<if type="motion_picture graphic broadcast" match="any">
+       <text variable="dimensions"/>
+   </if>
+   </choose>
+</macro>
+<macro name="originally-published">
+      <date variable="original-date" form="text" date-parts="year" prefix="[" suffix="] "/>
+  </macro>
+<macro name="version">
+<choose>
+<if type="book" match="any">
+       <text variable="version" prefix="Version "/>
+   </if>
+   </choose>
+</macro>
+<macro name="access">
+    <choose>
+      <if type="article article-journal article-magazine article-newspaper book broadcast chapter entry-dictionary entry-encyclopedia thesis graphic legal_case legislation manuscript map motion_picture paper-conference patent post post-weblog personal_communication report song speech webpage" match="any">
+      <text variable="URL" prefix="En ligne : " suffix=" "/>          
+    <text value="consulté le" prefix="[" suffix=" "/>
+       <date variable="accessed" form="text" suffix="">
+        <date-part name="day"/>
+            <date-part name="month"/>
+            <date-part name="year"/>
+          </date>
+      </if>
+    </choose>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="bill book broadcast graphic legal_case legislation map manuscript motion_picture patent personal_communication report song" match="any">
+        <text variable="title" text-case="capitalize-first" font-style="italic"/>
+      </if>
+      <else-if type="article-journal article-newspaper article-magazine" match="any">
+        <group delimiter=", ">
+          <text variable="title" text-case="capitalize-first" quotes="true" font-style="normal"/>
+          <text variable="container-title" font-style="italic"/>
+        </group>
+      </else-if>
+ <else-if type="post-weblog" match="any">
+        <group delimiter=", ">
+          <text variable="title" text-case="capitalize-first" quotes="true" font-style="normal"/>
+          <text variable="container-title" font-style="italic" prefix="Blog "/>
+        </group>
+      </else-if>
+      <else-if type="thesis" match="any">
+        <group delimiter=", ">
+          <text variable="title" text-case="capitalize-first" font-style="italic"/>
+          <text variable="genre"/>
+        </group>
+      </else-if>
+      <else-if type="manuscript" match="any">
+        <group delimiter=",">
+          <text variable="title" text-case="capitalize-first" quotes="true" font-style="normal"/>
+          <text variable="genre" prefix=" "/>
+        </group>
+      </else-if>
+      <else-if type="chapter entry-dictionary entry-encyclopedia" match="any">
+        <group>
+          <group suffix=" ">
```

Comment:
remove this group and set a `delimiter=" "` on the outer group for the spacing.

Reply:
i don't understand how to do that

Reply:
could you help me, please ?